commit f3fce92b27296068b4c304321b53bd1c7c4beb61
Author: lauren n. liberda <lauren@selfisekai.rocks>
Date:   Tue May 28 17:54:26 2024 +0000

    iwyu: missing <optional> in page_popup_controller.h
    
    Bug: 41455655
    Change-Id: I6483b9ec9d3c4acc08fb2bce8cac409c32a06dc4
    Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/5569043
    Reviewed-by: Lei Zhang <thestig@chromium.org>
    Commit-Queue: Lei Zhang <thestig@chromium.org>
    Reviewed-by: Khushal Sagar <khushalsagar@chromium.org>
    Cr-Commit-Position: refs/heads/main@{#1306871}

diff --git a/AUTHORS b/AUTHORS
index eba1dc976d8fa..715e108aca1c3 100644
--- a/AUTHORS
+++ b/AUTHORS
@@ -818,6 +818,7 @@ Lalit Chandivade <lalit.chandivade@einfochips.com>
 Lam Lu <lamlu@amazon.com>
 Laszlo Gombos <l.gombos@samsung.com>
 Laszlo Radanyi <bekkra@gmail.com>
+lauren n. liberda <lauren@selfisekai.rocks>
 Lauren Yeun Kim <lauren.yeun.kim@gmail.com>
 Lauri Oherd <lauri.oherd@gmail.com>
 Lavar Askew <open.hyperion@gmail.com>
diff --git a/third_party/blink/renderer/core/page/page_popup_controller.h b/third_party/blink/renderer/core/page/page_popup_controller.h
index 5da38d0b9ab26..c4e8e865fa96d 100644
--- a/third_party/blink/renderer/core/page/page_popup_controller.h
+++ b/third_party/blink/renderer/core/page/page_popup_controller.h
@@ -31,6 +31,8 @@
 #ifndef THIRD_PARTY_BLINK_RENDERER_CORE_PAGE_PAGE_POPUP_CONTROLLER_H_
 #define THIRD_PARTY_BLINK_RENDERER_CORE_PAGE_PAGE_POPUP_CONTROLLER_H_
 
+#include <optional>
+
 #include "third_party/blink/renderer/platform/bindings/script_wrappable.h"
 #include "third_party/blink/renderer/platform/heap/collection_support/heap_vector.h"
 #include "third_party/blink/renderer/platform/heap/garbage_collected.h"
